diff --git a/src/OpenCvSharpExtern/imgproc_Subdiv2D.h b/src/OpenCvSharpExtern/imgproc_Subdiv2D.h
index 38d1d9a..809526e 100644
--- a/src/OpenCvSharpExtern/imgproc_Subdiv2D.h
+++ b/src/OpenCvSharpExtern/imgproc_Subdiv2D.h
@@ -28,7 +28,7 @@ CVAPI(ExceptionStatus) imgproc_Subdiv2D_delete(cv::Subdiv2D *obj)
 CVAPI(ExceptionStatus) imgproc_Subdiv2D_initDelaunay(cv::Subdiv2D *obj, CvRect rect)
 {
     BEGIN_WRAP
-    obj->initDelaunay(rect);
+    obj->initDelaunay(cv::Rect(rect.x, rect.y, rect.width, rect.height));
     END_WRAP
 }
 
